feat(curriculum): add block 2 dialogs and tasks of the B1 course of the English Curriculum (#57471)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/24-b1-english-for-developers/learn-how-to-talk-about-past-experiences/671f7727d2e4240680c126bc.md

---
id: 671f7727d2e4240680c126bc
title: Task 99
challengeType: 19
dashedName: task-99
---

<!-- (Audio) Maria: I remember you would often stay late, fine-tuning the code until it was just right. Brian: Now I take care of the training so that others can do the same. -->

# --instructions--

Listen to the audio and answer the question below.

# --questions--

## --text--

What does Brian do now, and why does he do that?

## --answers--

He still fine-tunes the code himself, but not as often.

### --feedback--

Brian says he focuses on training others now, not doing the fine-tuning himself.

---

He takes care of hiring new team members for coding tasks.

### --feedback--

Brian mentions handling training, not hiring.

---

He reviews others' work without providing training.

### --feedback--

Brian emphasizes that he takes care of the training to help others learn.

---

He takes care of the training so that others can stay late to fine-tune the code.

## --video-solution--

4

# --explanation--

> The phrase `so that` is used to express purpose or reason, showing why an action is done. In this context, Brian takes care of the training with the purpose of enabling others to fine-tune code as he used to. Example: `She explained the process clearly so that everyone could understand it.` (this means her intent with explaining the process clearly was to make everyone understand it) 
> 
> In the dialog, Brian uses it to show that his goal in providing training is to empower others to perform at the same level as he did when he fine-tuned the code himself.

Reviewer: ```suggestion
The phrase `so that` is used to express purpose or reason, showing why an action is done. For example: 

`She explained the process clearly so that everyone could understand it.` - This means her intent with explaining the process clearly was to make everyone understand it. 

In the dialog, Brian uses it to show that his goal in providing training is to empower others to perform at the same level as he did when he fine-tuned the code himself.
```

Removed a section that was basically saying the same thing as the conclusion.
